src/resources/postcss/base/full/typography/_anchors.pcss

```diff
@@ -28,7 +28,7 @@
 
 		.tribe-theme-twentyseventeen &,
 		.tribe-theme-twentyseventeen .site-footer .widget-area &,
-		.theme-twentyseventeen .site-footer .widget-area & {
+		.site-footer .widget-area & {
```

Comment:
Isn't this too generic? It will affect non-tribe widgets: http://p.tri.be/Nt414y

Reply:
no, it has the `.tribe-common` wrapper around it (see line 1 of this file)